Training3/src/tasks/src/App.js

```diff
@@ -0,0 +1,30 @@
+import "./App.css";
+import { useEffect, useState } from "react";
+import Modal from "./Modal";
+
+function App() {
+  const [isOpen, setIsOpen] = useState(false);
+  function handleClick() {
+    setIsOpen(!isOpen);
+  }
+  useEffect(() => {
+    if (isOpen === true) {
+      const modal = document.querySelector(".Modal");
+      console.log(modal);
+      modal.addEventListener('keypress', (e) => {
```

Comment:
@phuongluongZigvy 
we don't add event listerner in real DOM, I think you should add to the document, and add listener when
- modal change to true
remove listener when
- modal change to false